lib/cloud_controller/deployment_updater/scheduler.rb

```diff
@@ -54,7 +52,6 @@ def update(update_frequency:, statsd_client:, prometheus_updater:)
           ##       so feed in the entire value!
           update_duration_ms = update_duration * 1000
           statsd_client.timing('cc.deployments.update.duration', update_duration_ms)
-          prometheus_updater.report_deployment_duration(update_duration_ms)
```

Comment:
This is being emitted on the scheduler VM. Therefore it can't be collected because there is no web server running.